README.md

```diff
@@ -120,15 +135,15 @@ NotImplementedError: Can't compare when one date falls within the other
 
 An `UndateInterval` is a date range between two `Undate` objects. Intervals can be open-ended, allow for optional labels, and can calculate duration if enough information is known
 ```python
->>> from undate.undate import UndateInterval
+>>> from undate import UndateInterval
 >>> UndateInterval(Undate(1900), Undate(2000))
 <UndateInterval 1900/2000>
->>> UndateInterval(Undate(1900), Undate(2000), label="19th century")
->>> UndateInterval(Undate(1900), Undate(2000), label="19th century").duration().days
-36890
-<UndateInterval '19th century' (1900/2000)>
->>> UndateInterval(Undate(1900), Undate(2000), label="20th century")
-<UndateInterval '20th century' (1900/2000)>
+>>> UndateInterval(Undate(1801), Undate(1900), label="19th century")
+>>> UndateInterval(Undate(1801), Undate(1900), label="19th century").duration().days
```

Comment:
Looking at these examples, my first question is are beginning and end values inclusive or exclusive? Looking up the date range for 19th century (which apparently I didn't know 😅), I realize it's inclusive but maybe worth pointing out?